### ISSUE TITLE
LIBEAY32.dll error when julia-studio starts on 64bit Win7

### ISSUE THREAD

**Issue by AntonioSaragga** [forio/julia-studio#192]
I'm getting the following sequential errors when Julia-studio (64bit Win7) starts:

```
The ordinal 4114 could not be located in the dynamic link library
LIBEAY32.dll

The procedure entry point EVP_PKEY_verify_init could not be located
in the dynamic link library LIBEAY32.dll
```

**Comment by keesvp** [comment]
I cannot reproduce this on my Windows system. I do have that dll, but if I remove it, I can start Julia Studio without problems. It's part of Intel iCLS Client, but we don't do anything with that. Could it be related with something peculiar of your set-up?

**Comment by keesvp** [comment]
OK, I take that back. That dll is existing in numerous other installations, and it is indeed necessary. It probably comes with windows, because we test the Julia Studio installation to run on 'bare' Windows machines. I suggest you try to find an installer for Intel iCLS Client to repair your system.

**Comment by AntonioSaragga** [comment]
Forgot to mention that this problem only appeared after I upgraded to julia-studio 0.4.4. Julia-studio 0.4.3 works fine. Also, after those two initial errors the IDE opens and works OK

**Comment by KrisKusano** [comment]
Sorry to revive this one @keesvp but I get a similar error pop-up when I launched for the first time (Windows 7, x64) using version 0.4.4 64 bit. The error is not present in 0.4.4 32 bit (without the DLL on the path).

```
The ordinal 354 could not be located in the dynamic link library LIBEAY32.dll
```

No second error.  I poked around and could not find an installer just for the Intel iCLS Client. I did, however, find the `LIBEAY32.dll` installed as part of TortoiseHg on my machine. I put a copy of the DLL on my path and still get the error.

The application does launch after I click OK on the error. I just installed, but Julia Studio appears to be functioning properly (writing code, submitting it).
